featuretools_tsfresh_primitives/utils.py

```diff
@@ -5,7 +5,7 @@
 
 
 def comprehensive_fc_parameters():
-    """It is basically a dictionary, which is a mapping from string to a dictionary list of parameters.
+    """A wrapper around :class:`ComperehensiveFCParameters` to filter out unsupported parameter settings.
```

Comment:
For the places where we reference `ComprehensiveFCParameters`, can we make it clear tha we are referring to a tsfresh function?

Reply:
Yes, updated the docstring.